src/OpenTelemetry.Instrumentation.Dependencies/SqlClientInstrumentationOptions.cs

```diff
@@ -22,12 +27,10 @@ namespace OpenTelemetry.Instrumentation.Dependencies
     /// </summary>
     public class SqlClientInstrumentationOptions
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="SqlClientInstrumentationOptions"/> class.
-        /// </summary>
-        public SqlClientInstrumentationOptions()
-        {
-        }
+        internal const string MicrosoftSqlServerDatabaseInstanceName = "db.mssql.instance_name";
+
+        private static readonly Regex DataSourceRegex = new Regex("^(.*?)(?:[\\\\,]|$)\\s*(.*?)(?:,|$)\\s*(.*)$", RegexOptions.Compiled);
```

Comment:
it would be handy to have an example of a string in a comment

Reply:
Done